### PR TITLE
[Documentation] update documentation as per API changes

### PR DESCRIPTION
**Pull request by iaulakh**
## Purpose

* update documentation

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Review by ahammer — DISMISSED** [review]

**Review by Mrayyan — DISMISSED** [review]

**Review by Mohtasim — DISMISSED** [review]

File: docs/CHANGELOG_UI_CALLING.md [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
 
 ### New Features
 - Call on hold
 - UI support for tablets
 
 ### Breaking Changes
 - CommunicationUIEventCode class is removed
-
+- Added "CallComposite" as a prefix to public classes
+- CallComposite.launch funcions are refactored. Argument classes GroupCallOptions, TeamsMeetingOptions are removed. New argument classes are added: CallCompositeRemoteOptions, CallCompositeLocalOptions
```
Comment (phileo99) [inline docs/CHANGELOG_UI_CALLING.md:12]: Spelling mistake:  should be "functions"

**Review by phileo99 — CHANGES_REQUESTED** [review]

File: docs/CHANGELOG_UI_CALLING.md (hunk 1 above)
Comment (phileo99) [reply docs/CHANGELOG_UI_CALLING.md]: also should be `launch()`

**Comment by phileo99** [comment]
Also @iaulakh  can we trying adding the tags for code:
https://www.markdownguide.org/cheat-sheet#:~:text=Extended%20Syntax%20%20%20%20Element%20%20,sentence%20with%20a%20footnote.%207%20more%20rows%20

**Review by ahammer — DISMISSED** [review]

**Review by ahammer — APPROVED** [review]

**Review by phileo99 — APPROVED** [review]
👍

**Pull request merged**